CHANGELOG.md

```diff
@@ -23,6 +23,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
   a generic rule
 - metavariable-pattern: Fix parse info when matching content of a metavariable
   under a different language
+- generic mode on Markdown files with very long lines will now work (#2987)
+
+### Changed
+- generic mode: files that don't look like nicely-indented programs
```

Comment:
I couldn't see any check being removed from the code, although perhaps it's a very small change and I'm missing it since there are many other changes to docs etc.

Reply:
It's the `--force` option that was added to the `spacegrep` command line... and the option in question was already implemented by spacegrep.